Combined

pimtegelaar opened the pull request.

pimtegelaar committed:
Added concept of combined results

pimtegelaar committed:
Merge remote-tracking branch 'origin/master' into combined

Commit:
Added more formatting.

Few changes made. The command used for the conversion is:

pandoc -f markdown -t latex combined.md -o combined.pdf -s --toc

There are still some comments in the source that need to be cleaned but I left
them there for the person who originally left them.

pimtegelaar committed:
Added abstract

pimtegelaar committed:
Removed reference from title

pimtegelaar committed:
Fixed spelling

pimtegelaar committed:
Merge remote-tracking branch 'origin/master' into combined

pimtegelaar committed:
Merge remote-tracking branch 'origin/master' into combined

pimtegelaar committed:
minor fix

alexvanmanen commented:
Looks nice. I would like it if our names and the current date is added. For the rest it looks very nice.

Next to that also our reflection needs to be uploaded, but that can be done in a different file.

softwaregast committed:
Added reflection Job

Anxys committed:
Spelling stuff & consistency

. .

Anxys committed:
Added a title and some formatting

Anxys committed:
Forgot s

Anxys committed:
Finishing touches

- Made a deliverable archive
- Added team name to document
- Compiled reflection to pdf

Anxys committed:
Changed combined name to final
